drivers/apc-epdu-mib.c

```diff
@@ -60,41 +60,48 @@ static snmp_info_t apc_epdu_mib[] = {
 	{ "device.version", ST_FLAG_STRING, SU_INFOSIZE, " .1.3.6.1.4.1.318.1.1.30.2.1.1.3.1", NULL, SU_FLAG_STATIC | SU_FLAG_OK, NULL },
 
 	/* Input */
+	/* ePDUPhaseTableSize = INTEGER: 1 */
+	{ "input.phases", 0, 1, ".1.3.6.1.4.1.318.1.1.30.3.0", NULL, SU_FLAG_OK, NULL },
 	/* ePDUDeviceStatusActivePower.1 = INTEGER: 785 */
 	{ "input.realpower", 0, 1, ".1.3.6.1.4.1.318.1.1.30.2.1.1.7.1", NULL, SU_FLAG_OK | SU_FLAG_NEGINVALID, NULL },
-	/* FIXME: Take first phase for global */
+	/* Take first phase for global if monophase */
 	/* ePDUPhaseStatusVoltage.1 = INTEGER: 2304 */
-	{ "input.voltage", 0, 0.1, ".1.3.6.1.4.1.318.1.1.30.4.2.1.4.1", NULL, SU_FLAG_OK | SU_FLAG_NEGINVALID, NULL },
-	/* ePDUPhaseTableSize = INTEGER: 1 */
-	{ "input.phases", 0, 1, ".1.3.6.1.4.1.318.1.1.30.3.0", NULL, SU_FLAG_OK, NULL },
+	{ "input.voltage", 0, 0.1, ".1.3.6.1.4.1.318.1.1.30.4.2.1.4.1", NULL, SU_FLAG_OK | SU_FLAG_NEGINVALID | SU_INPUT_1, NULL },
+	/* Take first phase for global if monophase */
```

Comment:
monophase => single phase or 1ph

Reply:
To take into account if new modification